core/test/unit/services/auth/session/store_spec.js

```diff
@@ -0,0 +1,255 @@
+const SessionStore = require('../../../../../server/services/auth/session/store'),
```

Comment:
General: A+++ for all these unit tests and avoiding routing tests ❤️

Reply:
Your unit tests look really clean 👍 Everything which should get mocked, is mocked. Niiice